collection.js

```diff
@@ -0,0 +1,41 @@
+const LOCAL_STORAGE_KEY = 'books';
+const list = document.getElementById('books');
+
+const getFromLocalStorage = () => {
+  const cachedBooks = window.localStorage.getItem(LOCAL_STORAGE_KEY);
+  return cachedBooks ? JSON.parse(cachedBooks) : [];
+};
+
+let booksArr = getFromLocalStorage();
+
+const saveToLocalStorage = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(booksArr));
+};
+
+function render() {
+  list.innerHTML = booksArr.map((book) => (`<li>
+            <article>
+                <p id="title">${book.title}</p>
+                <p id="author">${book.author}</p>
+                <button class="remove">Remove</button>
+            </article>`)).join('');
+
+  const remove = Array.from(document.getElementsByClassName('remove'));
+  remove.forEach((btn, i) => btn.addEventListener('click', () => {
+    booksArr = booksArr.filter((book, ind) => i !== ind);
+    saveToLocalStorage();
+    render();
+  }));
+}
+
+window.onload = render();
+
+const title = document.getElementById('newTitle');
+const author = document.getElementById('newAuthor');
+const addBtn = document.getElementById('add');
+
+addBtn.addEventListener('click', () => {
+  booksArr.push({ title: title.value, author: author.value });
+  saveToLocalStorage();
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20add%20a%20new%20book%20to%20the%20collection%2C%20with%20title%20and%20author.), Please Create a separate function to add a new book to the collection, with title and author and you can call it here in the event listener.